src/Inciter/Carrier.C

```diff
@@ -937,10 +1027,7 @@ Carrier::advance( uint8_t stage, tk::real newdt, uint64_t it, tk::real t )
   wait4app();
 
   // Advance stage in multi-stage time stepping by updating the rhs
-  if (m_stage < 1)
-    rhs( 0.5, m_u );
-  else
-    rhs( 1.0, m_uf );
+  rhs( m_stage < 1 ? m_u : m_uf );
```

Comment:
Do we gain anything from the ternary operator that offsets the loss in readability and clarity?